topo/bond/faucet/faucet.yaml

```diff
@@ -8,7 +8,7 @@ dps:
       priority: 1
     interfaces:
       4:
-        description: trunk
+        description: TESTING trunk
```

Comment:
where/how is this used?   The test you have is a unit test so don't use this bit.  What effect does putting TESTING here have and how does it manifest itself?

Reply:
My idea was to use it in the test where the Forch "testing_segment" was set to TESTING. But you are right it was not useful... I reverted the change and set "testing_segment" to "trunk".